src/test/java/uk/gov/hmcts/reform/unspec/utils/MonetaryConversionsTest.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.hmcts.reform.unspec.utils;
+
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class MonetaryConversionsTest {
+
+    @Test
+    public void shouldThrowNullPointer_whenGivenNullAmount() {
+        assertThrows(NullPointerException.class, () ->
+            MonetaryConversions.penniesToPounds(null));
+    }
+
+    @Test
+    public void zeroPenniesShouldConvertToZeroPounds() {
+        BigDecimal converted = MonetaryConversions.penniesToPounds(BigDecimal.ZERO);
+        assertThat(converted).isEqualByComparingTo("0");
+    }
+
+    @Test
+    public void zeroPoundsShouldConvertToZeroPennies() {
+        BigInteger converted = MonetaryConversions.poundsToPennies(BigDecimal.ZERO);
+        assertThat(converted).isEqualTo("0");
+    }
+
+    @Test
+    public void onePennyShouldConvertToOneHundredthOfPound() {
+        BigDecimal converted = MonetaryConversions.penniesToPounds(new BigDecimal("1"));
+        assertThat(converted).isEqualByComparingTo("0.01");
+    }
+
+    @Test
+    public void oneHundredthOfPoundShouldConvertToOnePenny() {
+        BigInteger converted = MonetaryConversions.poundsToPennies(new BigDecimal("0.01"));
+        assertThat(converted).isEqualTo("1");
+    }
+
+    @Test
+    public void tenPenniesShouldConvertToOneTenthOfPound() {
+        BigDecimal converted = MonetaryConversions.penniesToPounds(new BigDecimal("10"));
+        assertThat(converted).isEqualByComparingTo("0.10");
+    }
+
+    @Test
+    public void oneTenthofPoundShouldConvertToTenPennies() {
+        BigInteger converted = MonetaryConversions.poundsToPennies(new BigDecimal("0.10"));
+        assertThat(converted).isEqualTo("10");
```

Comment:
Minor: comparing string with BigInteger here. would be clean to compare same types instead. i.e. Same for few other places in this class.

```suggestion
        assertThat(converted).isEqualTo(BigInteger.valueOf(10));
```

Reply:
done